asylum-connect/control-panel
# WIP - add service area coverage filter

## Description
The service coverage area filter is now a dropdown filter that is opened by pushing the button which opens a dialog box with a form. The user's selected areas are then shown below the button.

## Asana ticket:
https://app.asana.com/0/1132189118126148/1199528219905008/f
## PR Checklist




## How to Test
1. Go into the control panel and press 'Select Service Areas' button
2. Add some service areas and press the 'Save' button in the dialog box
3. Add other filters if desired
4. Press the 'Search' button on the filters page
5. See if the organizations that populate the page have the correct service areas and other filter properties.
6. Verify that the search URL (found in 'Network' tab of Inspect in browser) is correct (has the correct search filter parameters).
*Example: "https://inreach-api-staging.herokuapp.com/v1/organizations?&page=1&serviceArea=service-national-united-states&properties=community-adults=true,&tagLocale=united_states&tags=Hygiene%20and%20Clothing.Clothes,Medical.Dental%20care"
*Note: make sure the 'serviceArea=' has the chosen service coverage areas.
